transcom/mymove
# MB-2110 TOO can view shipment weight details

## Description

> As a TOO
> When I view a shipment on the MTO tab
> Then I want to view the weights returned by the Prime for each shipment
>   So I can make informed decisions about accessorial service item approval

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

To test:
1. Log in as TOO, local
2. Click on any moves
3. Click on `Move task order` tab
4. Should see weights in the shipment card: 
![image](https://user-images.githubusercontent.com/1522549/90573361-22cf9080-e16b-11ea-8334-1f53a05db3b8.png)

Storybook:
1. `make storybook`
2. Navigate to http://localhost:6006/?path=/story/too-tio-components-shipmentweightdetails--with-no-details 

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2110) for this change

## Screenshots

![image](https://user-images.githubusercontent.com/1522549/90573377-2fec7f80-e16b-11ea-9b8a-f858d2c456d1.png)
